pkg/operators/formatters/formatters.go

```diff
@@ -66,26 +66,26 @@ const (
 	errnoTargetAnnotation     = "formatters.errno.target"
 )
 
-type formattersOperator struct{}
+type FormattersOperator struct{}
```

Comment:
Please undo this as indicated in https://github.com/inspektor-gadget/inspektor-gadget/pull/3502#discussion_r1783062319